docs/functional_architecture.rst

```diff
@@ -0,0 +1,70 @@
+.. _doc_functional_architecture:
+
+=========================================
+WiP : Parsec client functional components
+=========================================
+
+Missing translations and figures
+
+Mountpoint
+**********
+Le composant mountpoint est responsable de l’interaction avec le système de fichiers pour
```

Comment:
as `moutpoint` refers to the name of the component (and is not translated), it's maybe better to write it `Mountpoint`

Reply:
Changes it